CHANGELOG.md

```diff
@@ -6,6 +6,67 @@ This file is generated automatically. Add individual changelog entries to the 'c
+  [#1703](https://github.com/tenzir/vast/pull/1703)
+
+### :warning: Changes
+
+- The VAST community chat moved from Gitter to Slack. [Join us](http://slack.tenzir.com) in the `#vast` channel for interactive discussions.
+  [#1696](https://github.com/tenzir/vast/pull/1696)
+
+- The [tenzir/vast](https://hub.docker.com/r/tenzir/vast) Docker image now bundles the PCAP plugin.
+  [#1705](https://github.com/tenzir/vast/pull/1705)
+
+- VAST now merges lists from configuration files. E.g., running VAST with `--plugins=some-plugin` and `vast.plugins: [other-plugin]` in the configuration now results in `some-plugin` and `other-plugin` being loaded (sorted by the usual precedence).
+  [#1721](https://github.com/tenzir/vast/pull/1721)
+
+### :gift: Features
+
+- The new option `vast.start.commands` allows for specifying an ordered list of VAST commands that run after successful startup. The effect is the same as first starting a node, and then using another VAST client to issue commands.  This is useful for commands that have side effects that cannot be expressed through the config file, e.g., starting a source inside the VAST server that listens on a socket or reads packets from a network interface.
+  [#1699](https://github.com/tenzir/vast/pull/1699)
+
+- The options `vast.plugins` and `vast.plugin-dirs` may now be specified on the command line as well as the configuration. Use the options `--plugins` and `--plugin-dirs` respectively.
+  [#1703](https://github.com/tenzir/vast/pull/1703)
+
+- Add the reserved plugin name `bundled` to `vast.plugins` to enable load all bundled plugins, i.e., static or dynamic plugins built alongside VAST, or use `--plugins=bundled` on the command line. The reserved plugin name `all` causes all bundled and external plugins to be loaded, i.e., all shared libraries matching `libvast-plugin-*` from the configured `vast.plugin-dirs`.
+  [#1703](https://github.com/tenzir/vast/pull/1703)
+
+- It's now possible to configure the VAST endpoint as an environment variable by setting `VAST_ENDPOINT`. This has higher precedence than setting `vast.endpoint` in configuration files, but lower precedence than passing `--endpoint=` on the command-line.
+  [#1714](https://github.com/tenzir/vast/pull/1714)
+
+- Plugins now load their respective configuration from `<configdir>/vast/plugin/<plugin-name>.yaml` in addition to the regular configuration file at `<configdir>/vast/vast.yaml`. The new plugin-specific file does not require putting configuration under the key `plugins.<plugin-name>`. This allows for deploying plugins without needing to touch the `<configdir>/vast/vast.yaml` configuration file.
+  [#1724](https://github.com/tenzir/vast/pull/1724)
+
+### :beetle: Bug Fixes
+
+- Fixed a bug that would crash VAST when querying for non-strings in a string field with a partition synopsis.
+  [#1685](https://github.com/tenzir/vast/pull/1685)
```

Comment:
I think we should rewrite this to be more user-centric. What exactly does this mean for our users?